### PR TITLE
change build before test

### PR DESCRIPTION
**Pull request by yujinlim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
   "main": "lib/index.js",
   "types": "lib/src/index.d.ts",
   "scripts": {
-    "build": "webpack --env dev && webpack --env build && npm run test",
+    "build": "webpack --env dev && webpack --env build",
     "dev": "webpack --progress --colors --watch --env dev",
+    "pretest": "npm run build",
```
Comment (matiboy) [inline package.json:10]: 👍

File: package.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
   "main": "lib/index.js",
   "types": "lib/src/index.d.ts",
   "scripts": {
-    "build": "webpack --env dev && webpack --env build && npm run test",
+    "build": "webpack --env dev && webpack --env build",
```
Comment (matiboy) [inline package.json:8]: I don't quite understand what is wrong with running tests after build. But I have no problem not doing it either

**Review by matiboy — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (yujinlim) [reply package.json:8]: no no, there is no issue. there is a need to have run build before test on `npm test`, thats why i take it out and place it under pretest command, if you think we need to run test after build, i can restructure it.

Comment (matiboy) [reply package.json:8]: Nope, all good; approved PR

**Pull request merged**